src/utl/include/utl/Logger.h

```diff
@@ -194,6 +186,8 @@ class Logger
   void addSink(spdlog::sink_ptr sink);
   void addMetricsSink(const char *metrics_filename);
 
+  void setType(ToolId tool, int id, spdlog::level::level_enum level);
```

Comment:
'type' is so generic and overused.
spdlog calls it 'level' so why not use that terminology?

Reply:
You are reviewing draft PRs?  I don't plan to merge this as it stands.  I think we should only allow upgrade the level not downgrading.  If I do I agree setLevel is better.